extensions/amp-wordpress-embed/1.0/README.md

```diff
@@ -0,0 +1,174 @@
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-wordpress-embed
+```
+
+```javascript
+import '@ampproject/bento-wordpress-embed';
+```
+
+[/example]
+
+#### Example: Include via `<script>`
+
+[example preview="top-frame" playground="false"]
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+  <!-- These styles prevent Cumulative Layout Shift on the unupgraded custom element -->
+  <style data-bento-boilerplate>
+    bento-wordpress-embed {
+      display: block;
+      overflow: hidden;
+      position: relative;
+    }
+  </style>
+  <script async src="https://cdn.ampproject.org/v0/bento-wordpress-embed-1.0.js"></script>
+</head>
+<bento-wordpress-embed id="my-embed
+  data-url="https://make.wordpress.org/core/2015/10/28/new-embeds-feature-in-wordpress-4-4/"
+></bento-wordpress-embed>
+<div class="buttons" style="margin-top: 8px;">
+  <button id="switch-button">Switch embed</button>
+</div>
+
+<script>
+  (async () => {
+    const embed = document.querySelector('#my-embed');
+    await customElements.whenDefined('bento-wordpress-embed');
+
+    // set up button actions
+    document.querySelector('#switch-button').onclick = () => embed.setAttribute('data-url', 'https://make.wordpress.org/core/2021/09/09/core-editor-improvement-cascading-impact-of-improvements-to-featured-images/');
+  })();
+</script>
+```
+
+[/example]
+
+#### Layout and style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-wordpress-embed-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+  bento-wordpress-embed {
+    display: block;
+    overflow: hidden;
+    position: relative;
+  }
+</style>
+```
+
+**Container type**
+
+The `bento-wordpress-embed` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children (slides) via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties):
+
+```css
+bento-wordpress-embed {
+  height: 100px;
+  width: 100%;
+}
+```
+
+#### Attributes
+
+##### data-url (required)
+
+The URL of the post to embed.
+
+### Preact/React Component
+
+The examples below demonstrate use of the `<BentoWordPressEmbed>` as a functional component usable with the Preact or React libraries.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-wordpress-embed
+```
+
+```javascript
```

Comment:
I don't know if this makes a difference on the Github parser and/or the amp.dev parser, but maybe it should be JSX?

```suggestion
```jsx
```

Reply:
I've always wondered the difference here too. I updated it in the PR but not finding any definitive guidance online.